### ISSUE TITLE
User is not able to access Pie chart when Voiceover is turned ON: A11y_.NET 9 template_Home_Screen reader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27420]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.1; #A11ySev1; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #Element:Piechart; #Info and Relationship; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2 
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till "Pie chart"
4. Observe that User is not able to access Pie chart when Voiceover is turned ON.

### Actual Result:
User is not able to access Pie chart when Voiceover is turned ON.

### Expected Result:
Users should be able to access Pie chart when Voiceover is turned ON.

### User Impact:
Users who relay on screen reader wont get information about Pie chart.

### Version with bug

9.0.30 SR3

### Is this a regression from previous behavior?

No, this is something new

### Affected platforms

iOS

### Relevant log output

```shell

```

**Comment by PoojaNamde** [comment]
#Regressed:03-06-25;

Verified the issue in the latest environment, issue is still reproducing.

**Comment by merriemcgaw** [comment]
@tj-devel709 @PureWeen  is this one external to MAUI and in the specific SyncFusion controls? What are our next steps here?

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:07-28-25;

Closing this bug as logged new bug to track this issue.
Bug for reference: https://github.com/dotnet/maui/issues/30801
